src/Symfony/Component/Console/EventListener/ErrorListener.php

```diff
@@ -75,19 +75,15 @@ public static function getSubscribedEvents(): array
         ];
     }
 
-    private static function getInputString(ConsoleEvent $event): ?string
+    private static function getInputString(ConsoleEvent $event): string
     {
         $commandName = $event->getCommand()?->getName();
         $input = $event->getInput();
 
-        if ($input instanceof \Stringable) {
-            if ($commandName) {
-                return str_replace(["'$commandName'", "\"$commandName\""], $commandName, (string) $input);
-            }
-
-            return (string) $input;
+        if ($commandName) {
+            return str_replace(["'$commandName'", "\"$commandName\""], $commandName, (string) $input);
         }
 
-        return $commandName;
+        return (string) $input;
```

Comment:
I'm not sure the cast is necessary ?
`$input` implements `InputInterface` which as `__toString()` method. 
Thanks to PHP Type Juggling, `$input` will by stringified before returned.

Reply:
I'd prefer to keep the cast explicit tbh.

Reply:
> I'd prefer to keep the cast explicit tbh.

In this context, do you mean explicit for developers and/or explicit for PHP?